src/common/components/templates/InfoBanner.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { FaTimes } from "react-icons/fa";
+import * as Toast from "@radix-ui/react-toast";
+
+export default function InfoToast() {
+  const [isDisplayed, setIsDisplayed] = useState(false);
+  const [message, setMessage] = useState("");
+  const [duration, setDuration] = useState(10000);
+  const router = useRouter();
+  const { pathname, query } = router;
+  const userFarcasterName = query.handle;
+
+  useEffect(() => {
+    let storedState;
+
+    if (pathname === "/homebase") {
```

Comment:
can we move this check and the one on line 50 (in `closeToast`) to be a function called `isHomebasePage` so that when we update homebase paths to support tabs, the check will still work?

Reply:
sure, thanks for all those reviews !